modelling/ecological_mod/index.md

```diff
@@ -23,44 +23,107 @@ ____
+
+### Limitations
+
+ - Ocean Acidification is not paramterised. 
+ - Nutrient effects of algae of inshore reefs is not parameterised. 
+ - Long computational time. 
+
+Further information on ReefMod  can be found on [Github](https://github.com/ymbozec/REEFMOD.6.8_GBR){:target="\_blank"} 
 
 ![Figure10](../../assets/images/modelling/reefmod_figure_10.png)
 
-***Figure 2.** Schematic representation of the reef ecosystem model applied to the Great Barrier Reef (ReefMod-GBR). **(A)** Each of the 3,806 individual reefs is represented by a 20x20m horizontal space vertically colonized by coral colonies belonging to six morphological groups. **(B)** Demographic processes (solid arrows) and ecological interactions (dashed arrows) affecting coral colonies individually. **(C)** Modelling of crown-of-thorns starfish (CoTS) cohorts subject to size-specific survival during their life. For both corals and CoTS, settlement occurs from a pool of larvae that results from the retention of locally produced offspring (self-supply) and the incoming of larvae from connected reef populations (external supply) (Bozec et al., 2022).*
+***Figure 3.** Schematic representation of the reef ecosystem model applied to the Great Barrier Reef (ReefMod-GBR). **(A)** Each of the 3,806 individual reefs is represented by a 20x20m horizontal space vertically colonized by coral colonies belonging to six morphological groups. **(B)** Demographic processes (solid arrows) and ecological interactions (dashed arrows) affecting coral colonies individually. **(C)** Modelling of crown-of-thorns starfish (CoTS) cohorts subject to size-specific survival during their life. For both corals and CoTS, settlement occurs from a pool of larvae that results from the retention of locally produced offspring (self-supply) and the incoming of larvae from connected reef populations (external supply) (Bozec et al., 2022).*
 
 # References
 
-Bozec, Y. M., K. Hock, R. A. B. Mason, M. E. Baird, C. Castro-Sanguino, S. A. Condie, M. Puotinen, A. Thompson, and Mumby, P. J., 2022. Cumulative impacts across Australia’s Great Barrier Reef: A mechanistic evaluation. bioRxiv:1–37. [Read Here](https://esajournals.onlinelibrary.wiley.com/doi/epdf/10.1002/ecm.1494)
+Bozec, Y.M., Hock, K., Mason, R.A.B., Baird, M.E.,Castro-Sanguino, C., Condie, S.A., Puotinen, M., Thompson, A., & Mumby, P.J. (2022). Cumulative impacts across Australia’s Great Barrier Reef: A mechanistic evaluation. bioRxiv:1–37. https://doi.org/10.1002/ecm.1494
+
+Condie, S. and Condie, R. (2016). Retention of plankton within ocean eddies. Global Ecology and Biogeography. 25(10):1264-1277.https://doi.org/10.1111/geb.12485
+
+Condie, S., Plagányi-Lloyd, E., Morello, B., Hock, K., Beeden, R. (2018). Great Barrier Reef recovery through multiple interventions. Conservation Biology. 32(6):1356-1367. https://doi.org/10.1111/cobi.13161
+
+Condie, S.A., & Porobic, J. (2023). Coral Community Network (CoCoNet) Model: User Guide &Technical 
```

Comment:
"&Technical" --> "& Technical" ?

Reply:
updated

Reply:
oh, it still is showing up as "&Technical"